tests/libxc/dft-water-dimer/input.dat

```diff
@@ -0,0 +1,132 @@
+ 'MGGA_MVS':   -9.0564856780699685,    #TEST
+   'BHHLYP':   -9.3033999986561522,    #TEST
+  'wB97X-D':   -9.1757542124406264,    #TEST
+   'MPWB1K':   -9.0336915837231064,    #TEST
+    'TPSSh':   -8.8378235294361698,    #TEST
+ 'wB97X-D3':   -9.3288986589354810,    #TEST
+    'PWB6K':   -9.4282194193206958,    #TEST
+  'HCTH147':   -8.3162826882150807,    #TEST
+  'wB97X-V':   -9.4821424401823720,    #TEST
+    'O3LYP':   -7.8176167645711416,    #TEST
+  'HCTH407':   -8.5203608885543254,    #TEST
+    'B3P86':   -8.9141586168384173,    #TEST
+'SOGGA11-X':   -8.8745911935821375,    #TEST
+   'B3LYP5':   -9.0959603554256212,    #TEST
+   't-HCTH':   -8.7436651014659610,    #TEST
+   'B97M-V':   -8.8418491911446910,    #TEST
+      'M11':   -9.5734554263290548,    #TEST
+   'B3PW91':   -8.4004026584808535,    #TEST
+    'SOGGA':  -10.7798425028350557,    #TEST
+ 'B5050LYP':   -9.3793987409742403,    #TEST
+     'SCAN':   -9.5016966402125185,    #TEST
+     'PW91':  -10.0493750718829666,    #TEST
+    'B97-K':   -8.9959660891130131,    #TEST
+    'B97-D':   -8.8104921649044119,    #TEST
+  't-HCTHh':   -8.9869603237544453,    #TEST
+  'MPW1B95':   -9.0234377647582882,    #TEST
+'MGGA_MS2h':   -8.7155115739750109,    #TEST
+      'M06':   -8.9795216378895333,    #TEST
+   'MN12-L':   -7.6286366310499503,    #TEST
+  'LRC-BOP':  -10.1996778415893132,    #TEST
+      'M05':   -9.8289200658566269,    #TEST
+    'PBEOP':   -8.9222704948971234,    #TEST
+    'MPW1K':   -8.9485181505241194,    #TEST
+     'EDF1':   -7.0694470396180193,    #TEST
+  'wB97M-V':   -9.5515485049265028,    #TEST
+   'N12-SX':   -9.8132276848054865,    #TEST
+   'M05-2X':   -9.5470761819657408,    #TEST
+     'wB97':  -10.0529812434682917,    #TEST
+  'revPBE0':   -8.2364049959417738,    #TEST
+ 'MGGA_MS2':   -8.7028813137655785,    #TEST
+    'B97-1':   -9.2097800365755980,    #TEST
+    'B97-2':   -8.2846383254011364,    #TEST
+    'B97-3':   -8.2232124872273253,    #TEST
+   'B1PW91':   -8.2008207519650256,    #TEST
+   'wM05-D':   -9.2752390665725546     #TEST
+}                                      #TEST
+
+dimer = h2odimer.extract_subsets([1,2])                                 #TEST
+monA = h2odimer.extract_subsets(1)                                      #TEST
+monB = h2odimer.extract_subsets(2)                                      #TEST 
+for func in data.keys():                                                #TEST
+    try: dimer_energy = energy(func, molecule = dimer)                  #TEST
+    except: continue                                                    #TEST
+    monoA_energy = energy(func, molecule = monA)                        #TEST
+    monoB_energy = energy(func, molecule = monB)                        #TEST
+    func_ie = 627.5095 * (dimer_energy - monoA_energy - monoB_energy)   #TEST
+    try: compare_values(data[func], func_ie, 3, func)                   #TEST
+    except: continue                                                    #TEST
```

Comment:
`except: continue` looks like a Get Out of Jail Free line. is that really wanted?